src/main/java/com/uid2/shared/model/Site.java

```diff
@@ -0,0 +1,59 @@
+package com.uid2.shared.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class Site {
+    private final int id;
+    private final String name;
+    private Boolean enabled;
+    @JsonProperty("domain_names")
+    private Set<String> domainNames;
+
+    public Site(int id, String name, Boolean enabled) {
+        this.id = id;
+        this.name = name;
+        this.enabled = enabled;
+        this.domainNames = new HashSet<>();
+    }
+
+    public Site(int id, String name, Boolean enabled, Set<String> domains) {
+        this.id = id;
+        this.name = name;
+        this.enabled = enabled;
+        this.domainNames = domains;
+    }
+
+    public int getId() { return id; }
+    public String getName() { return name; }
+    public Boolean isEnabled() { return enabled; }
+    public Set<String> getDomainNames() { return domainNames; }
+
+    public void setEnabled(Boolean enabled) { this.enabled = enabled; }
+    public void setDomainNames(Set<String> domainNames) { this.domainNames = domainNames; }
```

Comment:
Do we need this? We're exposing `domainNames` through `getDomainNames`, so that might be enough for users of this class.

Reply:
Write operation to domain_names in admin is to "set" rather than "add/remove". Simpler to replace reference with the new set than to empty existing set and repopulate with new set.